backend/src/index.js

```diff
@@ -125,6 +125,65 @@ app.get(
   })
 )
 
+app.get(
+  '/set-status/:status/:id',
+  errorWrap(async (req, res) => {
```

Comment:
as a rule of thumb, any endpoint that modifies the database should be a POST or PUT request. Since this sets the status of a certain candidate, let's have the URL be something like `/candidates/:id/status` and make it a POST request